src/app/services/auth/auth.service.ts

```diff
@@ -1,41 +1,77 @@
 import { Injectable } from '@angular/core';
```

Comment:
Moving http logic into the service is a good idea.  I'm less happy that users were moved into this service.  The auth service just deals with authentication, so I don't want it to have a complete User with all the associated details.  The more decoupled auth and actual users are, the easier it will be to swap out this fake auth in the future.  Let's chat about the best way to implement this.